hack/gh-workflow-ci.sh

```diff
@@ -142,6 +144,11 @@ collect_logs() {
     done
     kubectl -n ${ns} get events >/tmp/logs/ns/${ns}/events
   done
+
+  # shellcheck disable=SC2038
+  find /tmp/logs -type f -exec grep -l "${test_gitea_smee_url}" {} \; | xargs sed -i "s|${test_gitea_smee_url}|SMEE_URL|g"
+  # shellcheck disable=SC2038
+  find /tmp/logs -type f -exec grep -l "${github_ghe_smee_url}" {} \; | xargs sed -i "s|${github_ghe_smee_url}|SMEE_URL|g"
```

Comment:
nit  you can use a loop

```suggestion
for url in "${test_gitea_smee_url}" "${github_ghe_smee_url}"; do
  # shellcheck disable=SC2038
  find /tmp/logs -type f -exec grep -l "${url}" {} \; | xargs sed -i "s|${url}|SMEE_URL|g"
  done
  ```